doc/source/methods/PermutationImportance.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Worth using a callout <code>Note</code> box for this? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>